app/src/main/java/com/framgia/fsalon/screen/homeadmin/AdminHomeViewModel.java

```diff
@@ -98,9 +100,9 @@ public void setPresenter(AdminHomeContract.Presenter presenter) {
     @Retention(RetentionPolicy.SOURCE)
     public @interface Tab {
         int TAB_SCHEDULER = 0;
-        int TAB_CUSTOMER = 1;
-        int TAB_RECEIPT = 2;
-        int TAB_REPORT = 3;
+        int TAB_RECEIPT = 1;
+        int TAB_REPORT = 2;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.